### PR TITLE
Add alerts for NCSO concessions

### PR DESCRIPTION
**Pull request by inglesp**
Note that this doesn't include code for sending the alert emails!

The form looks like this:

image

### CODE REVIEW

File: openprescribing/frontend/tests/test_spending_views.py [hunk 1]
```diff
@@ -107,6 +107,17 @@ def test_spending_views(self):
...
+        url = '/practice/{}/concessions/'.format(self.practice.code)
+        data = {
+            'email': 'alice@example.com',
+            'practice': self.practice.code,
+            'newsletters': ['alerts'],
+        }
+        response = self.client.post(url, data, follow=True)
+        self.assertContains(
+            response, "Check your email and click the confirmation link")
+
```
Comment (sebbacon) [inline openprescribing/frontend/tests/test_spending_views.py]: Can we test a `NCSOConcessionBookmark` is created here? Possibly also with the optional newsletter signup too?

File: openprescribing/templates/spending_for_one_entity.html [hunk 2]
```diff
@@ -33,6 +34,28 @@ <h1>{{ title }}</h1>
 
 <hr/>
 
+{% if form %}
+  {% if not signed_up_for_alert %}
+  <form method="post" class="form form-inline">
+   {% csrf_token %}
+   <p>
+     Get alerted by email when new price concession data is available.
```
Comment (sebbacon) [inline openprescribing/templates/spending_for_one_entity.html]: add something like "(up to 4 times per month)" (or whatever is accurate) so they know expected frequency?

**Comment by inglesp** [comment]
> I just want to check that I'm understanding correctly that this follows the pattern of the other alerts...

Yes, it all goes through `_handle_bookmark_and_newsletter_post`.

**Comment by inglesp** [comment]
I think this is now done, except that `send_ncso_concessions_alerts` will need to be manually called for the time being.

@evansd are you able to review this week?

**Comment by evansd** [comment]
@inglesp Have re-reviewed. Looks good :+1:

**Comment by inglesp** [comment]
Fixes #1366 and #1458.

**Pull request merged**